### ISSUE TITLE
SageMaker Component

### ISSUE THREAD

**Issue by pmuens** [serverless/components#439]
Most Data Scientist know how to work on the models but they don't know how to put them into production / setup an evironment to train them and use them in production for inference.

**Comment by eahefnawy** [comment]
Yeah, I was also thinking about an NLP component using AWS Comprehend! I've experimented with AWS Comprehend before and it was pretty easy. We could just add some high level http exposed functions to that.

**Comment by pmuens** [comment]
Nice! I like that idea.
